src/libopensc/reader-pcsc.c

```diff
@@ -2146,7 +2147,7 @@ part10_check_pin_min_max(sc_reader_t *reader, struct sc_pin_cmd_data *data)
 {
 	int r;
 	unsigned char buffer[256];
-	size_t length = sizeof buffer;
+	DWORD length = sizeof buffer;
```

Comment:
Again, when used as input for pcsc_internal_transmit this potentially leads to the wrong type